sig-architecture/annual-report-2021.md

```diff
@@ -87,51 +147,84 @@ In future, this will be generated from delta of sigs.yaml from $YYYY-01-01 to $Y
--
-
 Continuing:
-- [$subproject-name](https://git.k8s.io/community/$sig-id#$subproject-name)
--
 
-## Working groups
-
-<!--
-In future, this will be generated from delta of sigs.yaml from $YYYY-01-01 to $YYYY-12-31
-Manually visible via `git diff HEAD@{$YYYY-01-01} HEAD@{$YYYY-12-31} -- $sig-id/README.md`
--->
+- [Architecture and API Governance](https://github.com/kubernetes/community/tree/master/sig-architecture#architecture-and-api-governance-1)
+    - API guidance updates
+        - [Object references](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#object-references), including cross-namespace references from namespaced objects
+        - [Spec and Status](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#spec-and-status) ([#5842](https://github.com/kubernetes/community/pull/5842))
+    - 114 [API reviews completed](https://github.com/orgs/kubernetes/projects/13) in 2021 (30 for v1.21, 45 for v1.22, 39 for v1.23)
+    - Each SIG can identify 2-3 people to be involved in API reviews - https://github.com/kubernetes/kubernetes/blob/v1.23.0/OWNERS_ALIASES#L451-L452
+- [Conformance Definition](https://github.com/kubernetes/community/tree/master/sig-architecture#conformance-definition-1)
+    - We started 2021 with 128 endpoints remaining without conformance test.(69.13% conformance tested)
+    - The current count is [51 endpoints](https://apisnoop.cncf.io/conformance-progress/endpoints/1.24.0/?filter=untested) 
+      remaining without conformance tests putting us at [87.25%](https://apisnoop.cncf.io/?conformance-only=true) conformance tested.
+    - In the last 12 months
+	    - Tests for 75 endpoints was promoted to conformance 
+	    - 34 Endpoints was promoted to GA with conformance tests.
+	    - No new technical debt was incurred
+    - We moved the Conformance Office Hours Meeting from Bi-weekly to monthly in Aug 2021 to reduce the strain on attendee's calendars and increase engagement.
+    - At the start of 2021 the tracking of ineligible endpoints for conformance was moved to
+      [`ineligible_endpoints.yaml`](https://github.com/kubernetes/kubernetes/blob/master/test/conformance/testdata/ineligible_endpoints.yaml) file.
+      This allows for more community engagement on the eligibility of endpoints for conformance before any endpoint is added to the endpoint to the 
+      [Ineligible Endpoints list](https://apisnoop.cncf.io/conformance-progress/ineligible-endpoints) through pubic pull request.
+    - Our target for 2022 is to clean up the last 51 endpoint and then move over to Conformance profiles 
+      to further increase the value added by conformance testing.
+- [Code Organization](https://github.com/kubernetes/community/tree/master/sig-architecture#code-organization-1)
+     
+- [Enhancements](https://github.com/kubernetes/community/tree/master/sig-architecture#enhancement-proposals)
```

Comment:
Would it help to mention the reading club under the enhancements sections for discoverability purposes?

Reply:
@MadhavJivrajani yes please! propose a change and i'll gladly accept it.